src/lib/file-picker/file-picker.scss

```diff
@@ -1,11 +1,73 @@
-@use './mixins';
-
-@include mixins.styles;
+@use './core' as *;
 
 :host {
-  @include mixins.host;
+  @include tokens;
```

Comment:
Let's separate out the `:host` tokens selector to keep things easier to consume in the dev tools.